vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleRegistrationSpreadsheet.java

```diff
@@ -219,10 +219,6 @@ private void generateRowHeaderCell(int colIndex, int rowIndex) {
    * @param index 0-based row index of the row to remove
    */
   public void deleteRow(int index) {
-    if (getRows() == 1) {
-      // only one row remaining -> the header row
-      return;
-    }
```

Comment:
this needs to stay in. The last row can never be deleted (no matter what assumption is taken before).

Reply:
Good point. Do you think it makes sense to throw an exception here?

Reply:
no it is not that exceptional and I do not think anyone wants to delete the header